src/main/java/com/onarandombox/MultiversePortals/PortalPlayerSession.java

```diff
@@ -22,7 +22,7 @@
 import com.onarandombox.MultiversePortals.utils.PortalManager;
 
 public class PortalPlayerSession {
-    private MultiversePortals plugin;
+    private static MultiversePortals plugin;
```

Comment:
This shouldn't be static, any reason why you changed it to static?

Reply:
Yep, if I hadn't changed it to static, I couldn't use it in the `static void showStaticInfo()`, which I need to get the destination world.

Reply:
Remove static and see my comment regarding showStaticInfo method below.